.github/workflows/build.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches:
       - '**'
-  pull_request:
```

Comment:
I think this should have been in separate PR. Because, It's not related to the dungeon milestone.

Reply:
Fixed, that was accident merge between my fork master and this :P